src/hooks/useCioAutocomplete.ts

```diff
@@ -34,9 +34,10 @@ const useCioAutocomplete = (options: UseCioAutocompleteOptions) => {
     zeroStateSections,
     autocompleteClassName = 'cio-autocomplete',
     advancedParameters,
+    defaultValue,
```

Comment:
I think it would be better if we named this prop: `defaultInput` or `defaultQuery`